src/ui/theme.tsx

```diff
@@ -72,6 +72,8 @@ import { ComponentType } from "ui/tools/types/ComponentType";
 import type { Language } from "ui/i18n";
 import { createOnyxiaSplashScreenLogo } from "onyxia-ui/lib/SplashScreen";
 import { THEME_ID } from "keycloak-theme/login/envCarriedOverToKc";
+import { getPaletteOverride } from "./env";
+import _ from "lodash";
```

Comment:
I'd rather avoid introducing lodash.  
Please create a util in `src/ui/tools`. It's where we put all the general utilities.